cds_migrator_kit/rdm/migration/load/load.py

```diff
@@ -188,11 +209,25 @@ def _after_publish_mint_recid(self, record, entry, version):
             # but then we get a double redirection
             legacy_recid_minter(legacy_recid, record._record.parent.model.id)
 
+    def _after_publish_update_files_created(self, record, entry, version):
+        """Update the created date of the files post publish."""
+        # Fix the `created` timestamp forcing the one from the legacy system
+        # Force the created date. This can be done after publish as the service
+        # overrides the `created` date otherwise.
+        versions = entry.get("versions", {})
+        version_data = versions.get(version, {})
+        files = version_data.get("files", {})
+        for _, file_data in files.items():
+            file = record._record.files.entries[file_data["key"]]
+            file.model.created = arrow.get(file_data["creation_date"]).datetime
```

Comment:
I guess there is no way to create the file by the service layer and pass creation date in the dict? (in the records we had a similar limitation from the service schema but maybe this time we are lucky?)

Reply:
`created` and `updated` are [`dump_only`](https://github.com/inveniosoftware/invenio-records-resources/blob/713c1ce4c43c39b346bfd6f83b875143d8a2a65d/invenio_records_resources/services/files/schema.py#L123-L124), AFAIK there is no way to do it thorught the service layer.